app/services/hash_concatenator.rb

```diff
@@ -16,7 +16,7 @@ def concatenate
 
   def rename_keys(entries, replacing_string)
     entries.each do |entry|
-      entry['name'] = entry.delete('User')
+      entry['name'] = entry.delete('User').force_encoding('ISO-8859-1').encode('UTF-8')
```

Comment:
I'd expect this to happen right after the HTTP request happened. Then you can ensure that nobody works on the wrong encoding. Still I don't understand why this is necessary. JSON is standardized to use *UTF-8*.